### PR TITLE
Build MTE-335 [v114] Add build and test files to Bitrise UI Testing Blacklist

### PR DESCRIPTION
**Pull request by jjSDET**
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-335)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/FXIOS-14026)

### Description
Adding additional files to Bitrise UI Testing blacklist to save on costs of running unnecessary tests.

All files are now located across all directories and subdirectories which include:
- .yml/.yaml not named 'bitrise'.yml
- .sh
- .md
- .lproj
- everything in fastlane
- everything in taskcluster
- All Sync feature files
- All L10nSnapshotTests
- All test-fixture .py helpers

### Pull requests checks where applicable
- [x] Fill in the three TODOs above (tickets number and description of your work)
- [x] The PR name follows our [naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [x] Accessibility implemented and tested (minimum Dynamic Text and VoiceOver)
- [x] Unit tests written and passing
- [x] Documentation / comments for complex code and public methods

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -571,8 +571,23 @@ workflows:
...
+                #   .*\.lproj/.* -> matches all files whose parent directory begins with 'lproj'
+                #   fastlane/* -> matches all files in the fastlane directory
+                #   taskcluster/* -> matches all files in the taskcluster directory
+                #
+                # Features tested outside bitrise
+                #   Sync*/* -> matches all files in the Sync,SyncIntegrationTests,SyncTelemetry directories
+                #   L10nSnapshotTests/* -> matches all files in the L10nSnapshotTests directory
+                #
+                # Test Helpers
+                #   test-fixtures/.py$ -> matches all files whose file extension begine with 'py'
```
Comment (isabelrios) [inline bitrise.yml:589]: For test-fixtures, if the file is an .html file we should run the tests. Just double checking that that's the case here :) Thanks!

**Review by isabelrios — APPROVED** [review]
One question, apart from that, LGTM! Thanks

**Review by AaronMT — APPROVED** [review]
Look's OK to me. I say give it a go and monitor for any quirks 👍🏼

File: bitrise.yml (hunk 1 above)
Comment (jjSDET) [reply bitrise.yml:589]: yep! just included the `.py` files for now and didn't want to include any of the `.db` or `.html` files just in case.

**Pull request merged**